pr_testing/run-pr-unittests.sh

```diff
@@ -10,10 +10,10 @@ cd $WORKSPACE/${CMSSW_VERSION}
 CMSSW_PKG_COUNT=$(ls -d $LOCALRT/src/*/* | wc -l)
 REPORT_OPTS="--report-url ${PR_RESULT_URL} $NO_POST"
 
-rm -f ${RESULTS_DIR}/unittestGPU.txt
-mark_commit_status_all_prs 'unittests/gpu' 'pending' -u "${BUILD_URL}" -d "Running tests" || true
+rm -f ${RESULTS_DIR}/unittest${GPU_FLAVOR}.txt
+mark_commit_status_all_prs "unittests/${GPU_FLAVOR}" 'pending' -u "${BUILD_URL}" -d "Running tests" || true
 echo '--------------------------------------'
-mkdir -p $WORKSPACE/gpuUnitTests
+mkdir -p $WORKSPACE/${GPU_FLAVOR}UnitTests
 let UT_TIMEOUT=7200+${CMSSW_PKG_COUNT}*20
 UTESTS_CMD="USER_UNIT_TESTS=cuda timeout ${UT_TIMEOUT} scram b -v -k -j ${NCPU}  unittests "
```

Comment:
`USER_UNIT_TEST=cuda` also needs to be change for `rocm` tests

Reply:
Done